### PR TITLE
Add DinoPark Packs routes

### PR DESCRIPTION
**Pull request by gene1wood**
This adds routes for DinoPark Packs described in the hosted swagger file as well as routes present in the `sudo` app source code that aren't present in the swagger file.

Some of the descriptions of the `sudo` routes are missing as I couldn't determine by reading the code what they did.

I haven't marked any of the routes as internal or public as I'm unsure how to determine that.

### CODE REVIEW

File: Services.md [hunk 1]
```diff
@@ -59,6 +63,60 @@ Access Groups Management API – https://github.com/mozilla-iam/dino-park-packs
...
+- **/groups/api/v1/sudo/user/uuids/members** : GET : all member's UUIDs
+- **/groups/api/v1/sudo/user/uuids/staff** : GET : all staff's UUIDs
+- **/groups/api/v1/sudo/user/data/{userUuid}** : GET : all profile and group data for a given UUID
+- **/groups/api/v1/sudo/user/cis/{userUuid}** : POST : update CIS with a user's current groups
+- **/groups/api/v1/sudo/member/{groupName}** : POST : add member to group
+- **/groups/api/v1/sudo/member/{groupName}/{userUuid}** : DELETE : remove member from group
+- **/groups/api/v1/sudo/trust/groups/{groupName}** : PUT : 
+- **/groups/api/v1/sudo/groups/inactive** : GET : list inactive groups
+- **/groups/api/v1/sudo/groups/inactive/{groupName}** : DELETE : delete inactive group
+- **/groups/api/v1/sudo/groups/reserve/{groupName}** : POST :
```
Comment (fiji-flo) [inline Services.md]: reserves a group name for future use so it cannot be created (only if that name is deleted via DELETE inactive)

File: Services.md [hunk 2]
```diff
@@ -59,6 +63,60 @@ Access Groups Management API – https://github.com/mozilla-iam/dino-park-packs
...
+- **/groups/api/v1/sudo/user/{userUuid}** : DELETE : delete a user
+- **/groups/api/v1/sudo/user/inactive** : DELETE : delete inactive users
+- **/groups/api/v1/sudo/user/consolidate** : DELETE :
+- **/groups/api/v1/sudo/user/uuids/members** : GET : all member's UUIDs
+- **/groups/api/v1/sudo/user/uuids/staff** : GET : all staff's UUIDs
+- **/groups/api/v1/sudo/user/data/{userUuid}** : GET : all profile and group data for a given UUID
+- **/groups/api/v1/sudo/user/cis/{userUuid}** : POST : update CIS with a user's current groups
+- **/groups/api/v1/sudo/member/{groupName}** : POST : add member to group
+- **/groups/api/v1/sudo/member/{groupName}/{userUuid}** : DELETE : remove member from group
+- **/groups/api/v1/sudo/trust/groups/{groupName}** : PUT :
```
Comment (fiji-flo) [inline Services.md]: change the trust level of a group (e.g. from staff to nda) note that increasing the trust level will kick out members that don't have this trust level.

File: Services.md [hunk 3]
```diff
@@ -59,6 +63,60 @@ Access Groups Management API – https://github.com/mozilla-iam/dino-park-packs
...
+- **/groups/api/v1/sudo/user/uuids/staff** : GET : all staff's UUIDs
+- **/groups/api/v1/sudo/user/data/{userUuid}** : GET : all profile and group data for a given UUID
+- **/groups/api/v1/sudo/user/cis/{userUuid}** : POST : update CIS with a user's current groups
+- **/groups/api/v1/sudo/member/{groupName}** : POST : add member to group
+- **/groups/api/v1/sudo/member/{groupName}/{userUuid}** : DELETE : remove member from group
+- **/groups/api/v1/sudo/trust/groups/{groupName}** : PUT : 
+- **/groups/api/v1/sudo/groups/inactive** : GET : list inactive groups
+- **/groups/api/v1/sudo/groups/inactive/{groupName}** : DELETE : delete inactive group
+- **/groups/api/v1/sudo/groups/reserve/{groupName}** : POST : 
+- **/groups/api/v1/sudo/transfer** : POST :
```
Comment (fiji-flo) [inline Services.md]: transfer access groups from user a to user b. This is used for offboarding staff or people that lost their account.

File: Services.md [hunk 4]
```diff
@@ -59,6 +63,60 @@ Access Groups Management API – https://github.com/mozilla-iam/dino-park-packs
...
+- **/groups/api/v1/self/invitations/{groupName}** : DELETE : reject an invitation to a group
+- **/groups/api/v1/sudo/member/{groupName}** : POST : add a member
+- **/groups/api/v1/sudo/logs/all/raw** : GET : returns an audit log containing records of all actions taken on groups over time
+- **/groups/api/v1/sudo/mail/nda/{userUuid}** : POST : subscribe user to nda mailing list
+- **/groups/api/v1/sudo/mail/nda/{userUuid}** : DELETE : unsubscribe user to nda mailing list
+- **/groups/api/v1/sudo/curators/{groupName}** : GET : curators for a given group
+- **/groups/api/v1/sudo/curators/{groupName}** : POST : add a curator to a given group
+- **/groups/api/v1/sudo/user/{userUuid}** : DELETE : delete a user
+- **/groups/api/v1/sudo/user/inactive** : DELETE : delete inactive users
+- **/groups/api/v1/sudo/user/consolidate** : DELETE :
```
Comment (fiji-flo) [inline Services.md]: Sends all group members ships to CIS to consolidate (note: this treats DinoPark as source of truth).

File: Services.md [hunk 5]
```diff
@@ -59,6 +63,60 @@ Access Groups Management API – https://github.com/mozilla-iam/dino-park-packs
...
+- **/groups/api/v1/sudo/user/data/{userUuid}** : GET : all profile and group data for a given UUID
+- **/groups/api/v1/sudo/user/cis/{userUuid}** : POST : update CIS with a user's current groups
+- **/groups/api/v1/sudo/member/{groupName}** : POST : add member to group
+- **/groups/api/v1/sudo/member/{groupName}/{userUuid}** : DELETE : remove member from group
+- **/groups/api/v1/sudo/trust/groups/{groupName}** : PUT : 
+- **/groups/api/v1/sudo/groups/inactive** : GET : list inactive groups
+- **/groups/api/v1/sudo/groups/inactive/{groupName}** : DELETE : delete inactive group
+- **/groups/api/v1/sudo/groups/reserve/{groupName}** : POST : 
+- **/groups/api/v1/sudo/transfer** : POST :
+
```
Comment (fiji-flo) [inline Services.md:122]: - **/groups/api/v1/forms/email/** : GET : simple html form to send emails
- **/groups/api/v1/forms/email/bcc** : POST : send email on behalf of DinoPark
- **/groups/api/v1/forms/email/group** : POST : send email to a group

**Comment by gene1wood** [comment]
@fiji-flo Thanks so much for all of those additional details, I've incorporated all of them in 050c38e

**Pull request merged**